app/views/recipes/_recipeShow.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="recipe-details">
+
+  <div class="name">
+    <%= recipe.name %>
+    <%= button_to "Destroy this recipe", @recipe, method: :delete %>
+  </div>
+
+  <div class="details">
+    <div class="show-display">
+      <div class = "recipe-time">
+        <p>Preparation time: <%= recipe.preparation_time %> </p>
+        <p>Cooking time: <%= recipe.cooking_time %> </p>
+        <%= render "checkbox", recipe: recipe %>
+      </div>
+    </div>
+    <div class="steps">
+        <%= recipe.description %>
+    </div>
+    <div class="show-buttons">
+      <%= link_to 'Generate Shopping List', shopping_lists_path, class: 'generate-shopping-list' %>
```

Comment:
- [ ] Clicking the `Generate Shopping List` button on the show recipe page should generate an actual shopping list as per requirement. Please fix your code to achieve this.

Before Clicking the `Generate Shopping List` button
![image](https://github.com/lugard1/recipe_app/assets/81932478/8d082a8c-f8a8-4b2e-8c1c-d6e45f874f35)

After 
![image](https://github.com/lugard1/recipe_app/assets/81932478/3c9dae4f-dc7d-4147-b8e5-9a046399ea80)

#### :red_envelope: As you can see the shopping list is not being generated correctly :red_envelope: